x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/condition_entry_input/index.tsx

```diff
@@ -74,19 +80,31 @@ export const ConditionEntryInput = memo<ConditionEntryInputProps>(
       dataTestSubj,
     ]);
 
+    const getDropdownDisplay = (field: ConditionEntryField) => (
```

Comment:
Suggestion: move this function to be inside of the `useMemoI()` below, since it's only used there.

Also - I'm surprised you did not get a eslint error on line 114 for missing this dependency on the `useMemo()`.

Reply:
Sure, I will move it inside.
Why do you think it should get an eslint error?

Reply:
When using `useMemo()`, the second param to that is the dependencies that are used inside of the factory function - in this case, the `getDropdownDisplay` function should also be a dependency on it since its used.

Reply:
Totally right, but as the getDropdownDisplay wasn't intended to change I think it wasn't needed to put it in the dependencies array. Anyway, at the end was moved inside the useMemo 😅